### PR TITLE
Story1 

### PR DESCRIPTION
**Pull request by paulms1**
For review

### CODE REVIEW

File: be/dbcon.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+/*
+ * Creates connection to database
+ *
+ * @param N/A
```
Comment (charliecog) [inline be/dbcon.php]: if no params can just leave this line out

File: fe/index.php [hunk 2]
```diff
@@ -1,124 +1,32 @@
+<?php
```
Comment (charliecog) [inline fe/index.php]: this file (index.php) should be at the root of the project rather than in the fe directory

**Review by charliecog — COMMENTED** [review]
almost there - todo: update DB and remove joydivision image from tests

looks good though!

File: fe/index.php (hunk 2 above)
Comment (paulms1) [reply fe/index.php]: Cheers for saving me on this one!

File: be/dbcon.php (hunk 1 above)
Comment (paulms1) [reply be/dbcon.php]: done

**Review by charliecog — COMMENTED** [review]
Looks good - just need to delete collectionDB_2020-02-03.sql and data.txt

**Review by charliecog — APPROVED** [review]

**Pull request merged**